src/components/IOUConfirmationList.js

```diff
@@ -258,6 +321,28 @@ class IOUConfirmationList extends Component {
         ];
     }
 
+    /**
+     * Adds Venmo, if available, as the second option in the menu of payment options
+     */
+    addVenmoPaymentOptionToMenu() {
+        // Add Venmo option
+        if (this.props.localCurrencyCode === CONST.CURRENCY.USD && this.state.participants[0].phoneNumber.length > 0 && isValidUSPhone(this.state.participants[0].phoneNumber)) {
+            this.checkVenmoAvailabilityPromise = makeCancellablePromise(isAppInstalled('venmo'))
```

Comment:
Do you need to call `.promise` here like in the example? 
![2021-09-28_16-22-37](https://user-images.githubusercontent.com/42391420/135173826-5f320b97-6e41-4b83-b17f-abcb2c0d1ddd.png)

Reply:
yes we do, great catch